plugin/index.php

```diff
@@ -1,14 +1,18 @@
 <?php
-
 /**
  * Plugin Name: Gutenberg
  * Plugin URI: https://wordpress.github.io/gutenberg/
  * Description: Prototyping since 1440. Development plugin for the editor focus in core.
  * Version: 0.1
  */
-
+// Hook.
 add_action( 'admin_menu', 'gutenberg_menu' );
 
+/**
+ * Gutenberg's Menu.
+ *
+ * @since  0.0.1
```

Comment:
Do you think `@since` should follow the repository's own versioning, or that of what we expect the WordPress version to be upon integration? We may simply want to leave it out for future consideration.